bats/tests/containers/factory-reset.bats

```diff
@@ -86,16 +86,30 @@ check_installation() {
         # this one only exists after an update has been downloaded
         # ~/Library/Application Support/Caches/rancher-desktop-updater
     fi
+
+    if is_windows; then
+        delete_dir+=("$PATH_LOGS" "$PATH_DISTRO" "$PATH_DISTRO_DATA")
+    fi
+
     for dir in "${delete_dir[@]}"; do
         echo "$assert that $dir does not exist"
         ${assert}_not_exists "$dir"
     done
 
+    #Check if rancher-desktop WSL distros are deleted on Windows
+    if is_windows; then
+        run wsl --list
```

Comment:
I would make sure the command ran, and lists the current distro:

```bash
assert_success
assert_line "$WSL_DISTRO_NAME"
```

Reply:
Also, `wsl` must be called `wsl.exe` when called from inside a distro, and the output is in UTF16, so needs to be converted to utf8:

```console
jan@DESKTOP-4992680:~$ wsl --list
Command 'wsl' not found, but can be installed with:
sudo apt install wsl
jan@DESKTOP-4992680:~$ wsl.exe --list | grep $WSL_DISTRO_NAME
jan@DESKTOP-4992680:~$ wsl.exe --list | iconv -f utf16 | grep $WSL_DISTRO_NAME
Ubuntu-22.04 (Default)
```

So the command should be:

```suggestion
        run bash -c "wsl.exe --list | iconv -F utf16"
```

Reply:
I decided to invoke powershell instead of bash, as there's no need to convert the output. Tested this on windows 11, running the wsl command from inside the WSL distro.